src/pages/documentation/iam/tutorials/tabs/javascript-from-native.md

```diff
@@ -36,7 +36,9 @@ func shouldShowMessage(message: Showable) -> Bool {
 
 <Variant platform="android" task="call" repeat="4"/>
 
-With a reference to the `WebView`, the instance method `public void evaluateJavascript(@NonNull String script, @Nullable ValueCallback<String> resultCallback)` can now be leveraged to call a JavaScript method.
+With a reference to the `WebView`, the instance method `public void evaluateJavascript(@NonNull String script, @Nullable ValueCallback<String> resultCallback)` 
+
+can now be leveraged to call a JavaScript method.
```

Comment:
Is there a reason we split this across a few lines?

Reply:
nope... must have added a break accidentally